imap_processing/cdf/config/imap_ultra_l1a_variable_attrs.yaml

```diff
@@ -7,39 +7,42 @@ ultra_metadata_attrs:
   FORMAT: I19
   LABLAXIS: Metadata
   SCALE_TYP: linear
-  UNITS: " "
+  UNITS: ” ”
```

Comment:
Why is this changed?

Reply:
I think it's from copy and paste between slack and Mac. Good catch. I will change it back.